l3experimental/l3pdf/l3pdf.dtx

```diff
@@ -174,6 +174,27 @@
 %   This function may only be used up to the point where the PDF file is
 %   initialised.
 % \end{function}
+% \subsection{Destinations}
+% % \begin{function}[added = 2021-01-03]
+%   {\pdf_destination:nn}
+%   \begin{syntax}
+%     \cs{pdf_destination:nn} \Arg{name} \Arg{type or integer}
+%   \end{syntax}
+%   This creates a destination. \Arg{type or Integer} can be one of |fit|, |fith|,
+%   |fitv|, |fitb|, |fitbh|, |fitbv|, |fitr|, |xyz|
```

Comment:
We need to describe the meaning of the keywords, or at least reference the PDF spec, here.

Reply:
no problem, I can add it.